src/NerdMinerV2.ino.cpp

```diff
@@ -40,14 +42,32 @@ void alternate_screen_state() {
   //Serial.printf("Screen state is '%s', switching to '%s'", screen_state, !screen_state);
   Serial.println("Switching display state");
   digitalWrite(TFT_BL, !screen_state);
+  lastInteraction = millis();
 }
 
 void alternate_screen_rotation() {
   tft.getRotation() == 1 ? tft.setRotation(3) : tft.setRotation(1);
-
+  lastInteraction = millis();
 }
 
+void check_turn_off_screen() {
+  if (displayTimeout == 0) {
+    return;
+  }
 
+  unsigned long currentTime = millis();
+  unsigned long elapsedTime = currentTime - lastInteraction;
+
+  if (displayTimeout > 30000 && elapsedTime >= displayTimeout) {
+    int screen_state= digitalRead(TFT_BL);
```

Comment:
What about keeping the **same syntax style for variables**? In this case it would be `screenState` as we are using camelCase for variables everywhere else, no?

>**Note**: I see there are no "rules" for the overall project syntax style, but I think this could be something that we all could agree to keep a common syntax across the whole project. Ideally, we could install a linter that could automate this checking on the CI (for every commit-push), but for now, I just wanted to start creating awareness.

Reply:
Thank you. I agree on keeping variable names consistent. A linter would be a good idea to help with that.
In this case I just kept screen_state as it is, as it was already used that way in alternate_screen_state(). IMO it's better though to do variable renaming in a separate pull request or branch for clarity, so in this PR I want to keep it this way for now.

Reply:
Hi guys, all this functions like @alcar21 #95 Could be merged together. But I think will be cleaner if they are like inside an screen menu. There we could add more interesting thinks, like setting the initial screen, autobrightness level, change timezone... As an idea